pages/index.html

```diff
@@ -4,9 +4,39 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Olive Lavine</title>
+  <link rel="stylesheet" href="/stylesheets/home-style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Londrina+Outline&family=Press+Start+2P&display=swap" rel="stylesheet">
 </head>
 <body>
-  
+  <header>
+    <img src="/images/olive.jpg" alt="A black and white photo of a person 
+    from the shoulders up, with pale skin and straight long hair in a graduation 
+    cap and gown, against a background of clear skies and a tall city building" /> 
+    <nav>
+      <menu>
+        <li><a href="#">Home</a></li>
+        <li><a href="/pages/portfolio.html">Portfolio</a></li>
+        <li><a href="/pages/about.html">About Me</a></li>
+      </menu>
+    </nav>
+  </header>
+  <main> 
+    <!-- rosemary images sourced from freepik.com -->
```

Comment:
To get used to including it in our process, we should do our best to follow rules of attribution for outside resources at the time we use them in our project. The rules will vary by resource, but for freepik.com their rule is:
> If you are a Free user, you must always include the attribution line “Designed by Freepik” on the final products or designs you use our resources for, whether these are physical or digital products.

https://support.freepik.com/s/article/Attribution-How-when-and-where?language=en_US